### ISSUE TITLE
Implementation approach for ClientDB

### ISSUE THREAD

**Issue by march44** [attic-labs/noms#3749]
I am starting a mobile App project and was looking at ways to integrate noms into the App.

Are you planning on using [golang mobile](https://github.com/golang/mobile.git) to create a linkable  library for iOS and Android development?

I'd be happy to test a pre-release and give early feedback.

**Comment by aboodman** [comment]
Yeah that was what we were thinking on the client side, though we have not gotten started on it. Sorry the doc on the homepage wasn't clear, I'll fix that.
